src/openforms/validations/api/views.py

```diff
@@ -29,7 +29,11 @@ class ValidatorsListView(ListMixin, APIView):
     serializer_class = ValidationPluginSerializer
 
     def get_objects(self):
-        return list(register.iter_enabled_plugins())
+        param_component = self.request.query_params.get("component", None)
```

Comment:
It might be good to use a serializer here to validate the query params (similarly to, for example https://github.com/open-formulieren/open-forms/tree/master/src/openforms/appointments/api/views.py#L88)